modules/apps/object/object-web/types/src/main/resources/META-INF/resources/js/components/layout/layout-screen/AddNewTabButton.d.ts

```diff
@@ -12,7 +12,5 @@
  * details.
  */
 
-/// <reference types="react" />
```

Comment:
@kevenleone, I think that the error comes from this line, try to added just for testing